README.md

```diff
@@ -1 +1,272 @@
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [Video-presentation](#presentation)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [LEADERBOARD] 
+
+
+ **[LEADERBOARD]** website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. Build with HTML, CSS and Javascript.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML, CSS and Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">No Server build yet</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">No database build yet</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Used Linters]**
+- **[Great summary for the repo]**
+- **[Use of Github Flow]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](Not deployed yet)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+A new developer can get the project from my Github repo link i shall provide.
+
+To get a local copy up and running, follow these steps.
+
+
+go to https://github.com/stephenkati/Leaderboard.git
+
+clone or download the project
+
+Run it in your favourite browser
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A code editor, preferably VSCode. Any other code editor is okay.
+
+-Browser
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+run git clone https://github.com/stephenkati/Leaderboard.git
+
+or
+
+In the File menu, click Clone Repository.
+
+Click the tab that corresponds to the location of the repository you want to clone. You can also click URL to manually enter the repository location.
+
+Choose the repository you want to clone from the list.
+
+Click Choose... and navigate to a local path where you want to clone the repository.
+
+Click Clone.
+
+
+### Install
+
+Install this project with:
+
+Download the project from github and run it in your browser.
+
+
+### Usage
+
+To run the project, navigate to the project directory and use a live Server extension in your VSCode to run the project.
+
+
+### Run tests
+
+You can run tests to check for linter errors by running the following commands:
+
+- npx stylelint "**/*.{css,scss}" for testing css errors
+- npx eslint . for Javascript errors
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 update the instructions to use your app with webpack, the process is more complicated than cloning and running in the browser.

Reply:
Thanks for the comment, I will definitely add those instructions